src/views/profile.hbs

```diff
@@ -1,31 +1,33 @@
-<h1>profile</h1>
-<div id="user-options">
-  <a class="profile-links" href="/symptoms/home">
-    <br>
-    <br>
-    <img src="/assets/redcross.png" alt="">
-    <br> Add/Delete Symptoms
-  </a>
-  <a class="profile-links" href="/factors/home">
-    <br>
-    <br>
-    <img src="/assets/exercise.png" alt="">
-    <img src="/assets/sleep.png" alt="">
-    <img src="/assets/food.png" alt="">
-    <br>
-    <br>
-    <br> Add/Delete Factors</a>
-
-  <a class="profile-links" href="/addData">
-    <br>
-    <br>
-    <br>
-    <img src="/assets/scale.png" alt="">
-    <br>
-    <br> Enter Today's Data</a>
-  <a class="profile-links" href="/seeData">
-    <br>
-    <br>
-    <img src="/assets/smallGraph.png" alt="">
-    <br> See Data</a>
-</div>
\ No newline at end of file
+<section class="profile--section">
+  <h1 class="title">profile</h1>
+  <div class="profile--options">
+    <a class="profile--links" href="/symptoms/home">
+      <div class="profile--container-img">
+        <img src="/assets/redcross.png" alt="A plus icon">
```

Comment:
Hey Jennah, this page looks a ton nicer already. One thing, I'm not sure we want alt text on those icons. If anything, they should maybe be aria-hidden? I think they are more for decorative use so maybe we don't want them read aloud. Thoughts?

Reply:
I think you're right Tammy. I will remove the alt text and use an empty alt attribute instead. This link ([webAIM Alternate Text](https://webaim.org/techniques/alttext/) ) seems to suggest that an empty alt without aria-hidden is okay.

Reply:
Sounds perfect! Thanks. 

New problem: Travis failed for some reason.

Reply:
That's so strange... Travis seems to be passing now